### DISCUSSION TITLE
Replace deprecated features of `kustomize`

### DISCUSSION THREAD

**Discussion by Stringls** [fluxcd/flux2#4381] [Proposals]
@stefanprodan In relation to [the PR](https://github.com/fluxcd/website/pull/1704) to replace `patchesStrategicMerge` with inline patches. I'd really like to try to remove deprecated features of `kustomize` (e.g. `bases`, `vars` etc.)  for `flux2` repository - [Code search results](https://github.com/search?q=repo%3Afluxcd%2Fflux2+patchesStrategicMerge&type=code).
What do you think of that?

**Comment by stefanprodan** [comment]
That would be great! Thanks
